addons/stock/models/stock_picking.py

```diff
@@ -1733,12 +1733,16 @@ def _package_move_lines(self, batch_pack=False):
         move_line_ids = quantity_move_line_ids.filtered(lambda ml: ml.picked)
         if not move_line_ids:
             move_line_ids = quantity_move_line_ids
+        if self.env.context.get('move_lines_to_pack_ids', False):
+            move_line_ids = move_line_ids.filtered(lambda ml: ml.id in self.env.context['move_lines_to_pack_ids'])
```

Comment:
There is no point to keep those lines once you use a arg instead of the context

Reply:
You're right. Should I create another PR to remove these lines?

Reply:
@mohammadhossam Yep, you can send me the PR once you created it